<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projector_dlg</class>
 <widget class="QDialog" name="projector_dlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="log_tedt">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connect_btn">
      <property name="text">
       <string>&amp;Connect</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quit_btn">
      <property name="text">
       <string>&amp;Quit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
